tests/telemetry/test_telemetry.py

```diff
@@ -164,29 +164,42 @@ def test_telemetry_queue_buffer_cnt(duthosts, enum_rand_one_per_hwsku_hostname,
     logger.info('start telemetry output testing')
     dut_ip = duthost.mgmt_ip
 
+    interfaces_status = duthost.show_and_parse("show interface status")
```

Comment:
We can use API "duthost.get_interfaces_status()"

Something like this

   interfaces = duthost.get_interfaces_status()
   pattern = re.compile(r'^Ethernet[0-9]{1,2}$')
   interface_list = []
   for interface, status in interfaces.items():
       if pattern.match(interface) and status["oper"] == "up" and status["admin"] == "up":
           interface_list.append(interface)

Reply:
Thanks for the comment, due to internal review I modified it to import and use ports_list as it returns only up ports.